src/components/Proposal/RedemptionsTip.tsx

```diff
@@ -83,9 +83,13 @@ export default (props: IProps) => {
             </li> : ""
           }
           {contributionRewards["rep"] ? <li><Reputation reputation={contributionRewards["rep"]} totalReputation={dao.reputationTotalSupply} daoName={dao.name} /></li> : ""}
-          { /*
-            TOOD: add native token
-          */ }
+
+          {contributionRewards["nativeToken"] ?
```

Comment:
weird to add a new feature in a hack PR that we will hopefully roll back soon

Reply:
it is not the whole PR that is is a hack, the PR was all nice and dandy until the last commit which contains the hack.